app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,128 @@
+module Types
+  class QueryType < Types::BaseObject
+    field :comments, Types::CommentType.connection_type, "Returns all comments", null: false
+    field :comment, Types::CommentType, "Returns comment for ID", null: false do
+      argument :id, ID, required: true, default_value: false
+    end
+
+    field :debates, Types::DebateType.connection_type, "Returns all debates", null: false
+    field :debate, Types::DebateType, "Returns debate for ID", null: false do
+      argument :id, ID, required: true, default_value: false
+    end
+
+    field :geozones, Types::GeozoneType.connection_type, "Returns all geozones", null: false
+    field :geozone, Types::GeozoneType, "Returns geozone for ID", null: false do
+      argument :id, ID, required: true, default_value: false
+    end
+
+    field :proposals, Types::ProposalType.connection_type, "Returns all proposals", null: false
+    field :proposal, Types::ProposalType, "Returns proposal for ID", null: false do
+      argument :id, ID, required: true, default_value: false
+    end
+
+    field :proposal_notifications, Types::ProposalNotificationType.connection_type, "Returns all proposal notifications", null: false
+    field :proposal_notification, Types::ProposalNotificationType, "Returns proposal notification for ID", null: false do
+      argument :id, ID, required: true, default_value: false
+    end
+
+    # Support deprecated API
+    field(
+      :proposal_notifications,
+      Types::ProposalNotificationType.connection_type,
+      "Returns all proposal notifications",
+      null: false,
+      camelize: false,
+      deprecation_reason: "Snake case fields are deprecated. Please use proposalNotifications."
+    )
+
+    field(
+      :proposal_notification,
+      Types::ProposalNotificationType,
+      "Returns proposal notification for ID",
+      null: false,
+      camelize: false,
+      deprecation_reason: "Snake case fields are deprecated. Please use proposalNotification."
+    ) do
+      argument :id, ID, required: true, default_value: false
+    end
```

Comment:
Are these lines necessary now that we've changed the `field` method to generate the snak case field as well? :thinking:

Reply:
Yup, can confirm that this works!

Reply:
Removed in [e155403](https://github.com/consul/consul/pull/4766/commits/e155403aa55b9b0cf2a2ce208bbaacbaedd17566)